.azure-pipelines/run-test-elastictest-template.yml

```diff
@@ -338,10 +326,12 @@ steps:
             echo -n "$(FRONTEND_URL)/scheduler/testplan/$TEST_PLAN_ID "
             echo -e "\033[33mfor detailed test plan progress \033[0m"
             # When "EXECUTING" finish, it changes into "KVMDUMP", "FAILED", "CANCELLED" or "FINISHED"
-            echo "[test_plan.py] poll EXECUTING status"
-            python ./.azure-pipelines/test_plan.py poll -i $TEST_PLAN_ID --expected-state EXECUTING --expected-result ${{ parameters.EXPECTED_RESULT }}
+            echo "[test_plan.py] poll EXECUTING status, timeout 22 hours"
```

Comment:
Looks like the skip logic in the cancel test plan stage is enough, why we need to change this?